pynest/nest/lib/hl_api_simulation.py

```diff
@@ -139,8 +144,19 @@ def RunManager():
     ::
 
         with RunManager():
-            for i in range(10):
-                Run()
+            for _ in range(10):
+                Run(100)
+                # extract results
+
+    Notes
+    -----
+
+    Be careful about modifying the network or neurons inside the `RunManager` context,
+    especially not to `Create` and `Connect` neurons or call `SetKernelStatus`. Calling
+    `SetStatus` to change membrane potential `V_m` of neurons or synaptic weights
+    (but not delays!) will in most cases work as expected, while changing membrane or
+    synaptic times constants **will not work correctly**. If in doubt, assume that changes
+    may cause undefined behavior and check these thoroughly.
```

Comment:
```suggestion
    ```suggestion
    Be careful about modifying the network or neurons inside the `RunManager` context.
    In particular, do not call `Create`, `Connect`, or `SetKernelStatus`. Calling `SetStatus`
    to change membrane potential `V_m` of neurons or synaptic weights (but not delays!)
    will in most cases work as expected, while changing membrane or synaptic times
    constants **will not work correctly**. If in doubt, assume that changes may cause
    undefined behavior and check these thoroughly.
```